.github/workflows/build.yml

```
@@ -230,12 +236,14 @@ jobs:
       - name: Run Django Migrations and Collectstatic Jobs
         run: |
           kubectl delete job django-migrate-job --ignore-not-found
+          kubectl wait --for=delete job/django-migrate-job --timeout=60s
           kubectl apply -f k8s/jobs/django-migrate-job.yml
 
           echo "Waiting for job to complete..."
           kubectl wait --for=condition=complete job/django-migrate-job --timeout=120s
           
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)